src/index.html

```diff
@@ -1,15 +1,190 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <meta http-equiv="X-UA-Compatible" content="ie=edge">
-  <title>HTML Form</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>HTML Form</h1>
-  <script type="text/javascript" src="./main.js"></script>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>HTML Form</title>
+    <link rel="stylesheet" href="./style.css">
+  </head>
+  <body>
+    <script type="text/javascript" src="./main.js"></script>
+    <form
+      action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+      method="POST">
+      <fieldset class="set">
+        <legend>Personal information</legend>
+        <label class="form-field" for="Surname">
+          Surname:
+          <input
+            type="text"
+            name="Surname"
+            id="Surname"
+            autocomplete="off"
+          >
+        </label>
+        <br>
+        <label class="form-field" for="name">
+          Name:
+          <input
+            type="text"
+            name="Name"
+            id="name"
+            autocomplete="off"
+          >
+        </label>
+        <br>
+        <label class="form-field" for="age">
+          How old are You?
+          <input
+            type="number"
+            name="age"
+            id="age"
+            value="12"
+            min="0"
+            max="300"
+          >
+        </label>
+        <br>
+        <label class="form-field" for="birth">
```

Comment:
```suggestion
        <label class="form-field" for="birthdate">
```
It's not clear what 'birth' means